scripts/qa_crowbarsetup.sh

```diff
@@ -1233,12 +1241,23 @@ function create_repos_yml()
         SUSE-Enterprise-Storage-2-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
 
-    create_repos_yml_for_platform "suse-12.1" "x86_64" "$tftpboot_repos12sp1_dir" \
-        SLES12-SP1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/ \
-        SLE12-SP1-HA-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/ \
-        SUSE-OpenStack-Cloud-6-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/ \
-        SUSE-Enterprise-Storage-2.1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2.1:/x86_64/update/ \
-        >> $tmp_yml
+    if iscloudver 6; then
```

Comment:
About these two blocks, the only diffrence is the version of storage repo. Maybe you can first create $tmp_yml with non-storage stuff, and then add a right version of storage repos according to product version?

Also, maybe it would be good to have a variable at the top declaring which SES version are we gonna use, and later check against it and not just against SOC version. (We might even want to support SES3 with SOC6 in the future)